src/components/Dialog/PresetManageDialog.vue

```diff
@@ -1,51 +1,166 @@
+                            {{ parameterLabels[sliderKey as ParameterType] }}
+                          </span>
+                          <span>
+                            {{
+                              selectedPreset[
+                                sliderKey as ParameterType
+                              ].toFixed(2)
+                            }}
+                          </span>
+                        </div>
+                        <!-- NOTE: プルリクエスト分割のため、まずは動かないスライダーが実装されている。変更機能も実装予定。 -->
+                        <BaseSlider
+                          :min="value.min()"
+                          :max="value.max()"
+                          :step="value.step()"
+                          :modelValue="
+                            selectedPreset[sliderKey as ParameterType]
+                          "
+                        />
+                      </div>
+                    </template>
+                  </div>
+                  <div
+                    v-if="selectedPreset.morphingInfo"
+                    class="parameter-list"
+                  >
+                    <h3 class="parameter-headline">モーフィング</h3>
+                    <div class="mophing-style">
+                      <CharacterButton
+                        v-if="morphingTargetCharacterInfo"
+                        :selectedVoice="{
+                          engineId: selectedPreset.morphingInfo.targetEngineId,
+                          speakerId:
+                            selectedPreset.morphingInfo.targetSpeakerId,
+                          styleId: selectedPreset.morphingInfo.targetStyleId,
+                        }"
+                        :characterInfos="[morphingTargetCharacterInfo]"
+                        :showEngineInfo="morphingTargetEngines.length >= 2"
+                        :emptiable="true"
+                        :uiLocked="false"
+                      />
+                      <span v-if="morphingTargetCharacterInfo">
+                        {{ morphingTargetCharacterInfo.metas.speakerName }}
+                      </span>
+                      <span
+                        v-if="
+                          morphingTargetCharacterInfo &&
+                          morphingTargetCharacterInfo.metas.styles.length >= 2
+                        "
+                      >
+                        （{{ morphingTargetStyleInfo?.styleName }}）
+                      </span>
+                    </div>
+                    <div class="parameter">
+                      <div class="parameter-header">
+                        <span>割合</span>
+                        <span>
+                          {{ selectedPreset.morphingInfo.rate.toFixed(2) }}
+                        </span>
+                      </div>
+                      <!-- NOTE: プルリクエスト分割のため、まずは動かないスライダーが実装されている。変更機能も実装予定。 -->
+                      <BaseSlider
+                        :min="0"
+                        :max="1"
+                        :step="0.01"
```

Comment:
まだ動かないので関係ないですが、正しくはこうかも？
```suggestion
                        :min="SLIDER_PARAMETERS.morphingRate.min()"
                        :max="SLIDER_PARAMETERS.morphingRate.max()"
                        :step="SLIDER_PARAMETERS.morphingRate.step()"
```